src/map.h

```diff
@@ -189,6 +189,13 @@ class Map /*: public NodeContainer*/
 	// Returns NULL if not found
 	MapBlock * getBlockNoCreateNoEx(v3s16 p);
 
+	// Overridden by server
+	virtual bool blockNotInDatabase(v3s16 pos)
```

Comment:
This doesn't really matter.  It will be 8b on 64-bit systems (probably > 90% of where Minetest is used outside of Android) and the difference isn't big enough to matter either way.